examples/lwt/lwt_h2c.ml

```diff
@@ -105,8 +105,8 @@ let connection_handler =
         Status.to_string error
     in
     let body = handle Headers.empty in
-    Body.Writer.write_string body message;
-    Body.Writer.close body
+    H2.Body.Writer.write_string body message;
```

Comment:
Please revert this file, it's supposed to be `Httpaf.Body`.

Reply:
done